services/worker-manager/src/api.js

```diff
@@ -103,7 +102,14 @@ builder.declare({
     'worker-manager:provider:<providerId>',
   ]},
   description: [
-    'Given an existing worker pool definition, this will modify it and return the new definition.',
+    'Given an existing worker pool definition, this will modify it and return',
+    'the new definition.',
+    '',
+    'To delete a worker pool, set its `providerId` to `"null"` and empty its',
+    'configuration (note that this is a string and not the JSON value `null`).',
```

Comment:
We could also make `null-config` allow any properties and then you wouldn't need to delete the config?

Reply:
Oh, that's a good idea.